WooCommerce/Resources/WooCommerce.storekit

```diff
@@ -0,0 +1,54 @@
+{
+  "identifier" : "28D2E099",
+  "nonRenewingSubscriptions" : [
+
+  ],
+  "products" : [
+
+  ],
+  "settings" : {
+    "_applicationInternalID" : "1389130815",
+    "_developerTeamID" : "PZYM8XX95Q",
+    "_lastSynchronizedDate" : 689685986.85126996
```

Comment:
I'm having second thoughts about this. I added the StoreKit configuration with sync enabled, but if we're only going to use this for testing, maybe I should convert to local and ensure it only has the debug product? I'm not sure we want the whole product catalog synced to a public repo.

Reply:
Yeah, if it's only for unit testing I would convert it to local and only add the debug product, no need to have the whole product catalog even if I don't see huge security issues with that.